### DISCUSSION TITLE
Chat push notification

### DISCUSSION THREAD

**Discussion by hidayet-poscu** [serverpod/serverpod#2558] [Q&A] [answered]
I want to use the Serverpod chat package. Does Serverpod have a solution for messages to be delivered when the application is closed?

**Comment by Isakdl** [comment, marked as answer]
We do not have an out of the box push notification solution today, issue is tracked here: https://github.com/serverpod/serverpod/issues/439.

But with that said using FCM with the REST API and/or a client like this: https://pub.dev/packages/firebaseapis should allow you to integrate native push notifications on android / ios / web and send them from the server.

Converting this to a discussion as we already have an issue for push notifications.
